src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostOptions.cs

```diff
@@ -36,7 +36,7 @@ internal static class RemoteHostOptions
         // normally response time is within 10s ms. at most 100ms. if priority is changed to "Normal", most of time 10s ms.
         [ExportOption]
         public static readonly Option<int> RequestServiceTimeoutInMS = new Option<int>(
-            nameof(InternalFeatureOnOffOptions), nameof(RequestServiceTimeoutInMS), defaultValue: 10 * 60 * 1000,
+            nameof(InternalFeatureOnOffOptions), nameof(RequestServiceTimeoutInMS), defaultValue: 7 * 24 * 60 * 60 * 1000 /* 7 days */,
```

Comment:
:bulb: Since there is no regression test, it would be good to have a comment explaining why the timeout is so long.

Reply:
sure